projects/vdk-plugins/vdk-structlog/src/vdk/plugin/structlog/formatters.py

```diff
@@ -42,6 +42,23 @@ def add_fields(
             log_record["level"] = record.levelname
 
 
+class LtsvFormatter(Formatter):
+    """
+    Formats logs in the LTSV format.
```

Comment:
I would like if the not acronym value was used in the comment

Reply:
I agree here